examples/basic/plasma/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+plasma_store -m 50000000 -s /run/oremda/plasma.sock
```

Comment:
FYI, there is no way to stop this cleanly with singularity, because there isn't a straightforward way to stop/kill the singularity processes. But if we move running the plasma store to the pipeline code (via the context manager function), it will be able to stop cleanly.